baserun/instrumentation/anthropic.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import logging
+from types import GeneratorType
+from typing import Collection, Any
+
+from anthropic.resources import Completions, AsyncCompletions
+from anthropic.types import Completion
+from opentelemetry.sdk.trace import Span
+
+from baserun.instrumentation.base_instrumentor import BaseInstrumentor
+from baserun.instrumentation.span_attributes import (
+    SpanAttributes,
+    ANTHROPIC_VENDOR_NAME,
+)
+
+logger = logging.getLogger(__name__)
+
+_instruments = ("anthropic >= 0.3.0",)
+__version__ = "0.1.0"
+
+
+class AnthropicInstrumentor(BaseInstrumentor):
+    """An instrumentor for OpenAI's client library."""
+
+    WRAPPED_METHODS = [
+        {
+            "class": Completions,
+            "function": Completions.create,
+            "span_name": "anthropic.completion",
+        },
+        {
+            "class": AsyncCompletions,
+            "function": AsyncCompletions.create,
+            "span_name": "anthropic.completion",
+        },
+    ]
+
+    def instrumentation_dependencies(self) -> Collection[str]:
+        return _instruments
+
+    @staticmethod
+    def set_request_attributes(span: Span, kwargs: dict[str, Any]):
+        span.set_attribute(SpanAttributes.LLM_VENDOR, ANTHROPIC_VENDOR_NAME)
+        span.set_attribute(SpanAttributes.LLM_REQUEST_MODEL, kwargs.get("model"))
+        span.set_attribute(
+            SpanAttributes.LLM_CHAT_STOP_SEQUENCES, kwargs.get("stop_sequences")
```

Comment:
Why does this have CHAT in the name?

Reply:
I got the attributes from here https://github.com/traceloop/openllmetry/tree/main/packages/opentelemetry-semantic-conventions-ai